<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeWidget</class>
 <widget class="QWidget" name="UpgradeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>终端升级</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TRES.qrc">
    <normaloff>:/new/prefix1/Tres/Gear.ico</normaloff>:/new/prefix1/Tres/Gear.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>420</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="terAddrClrBttn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>清空终端地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="terAddrABttn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>终端地址指定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uStartBttn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>升级开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uPauseBttn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>暂停升级</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="TRES.qrc"/>
 </resources>
 <connections/>
</ui>
